### DISCUSSION TITLE
Provide documentation about annotations

### DISCUSSION THREAD

**Discussion by kl0ck** [eclipse/windowbuilder#434] [Q&A]
I know there are many useful javadoc annotations that can be used on code to solve some issues with WindowBuilder. 
The problem is it's very difficult to find help about these annotations (or tags). Please provide documentation about them.

Some useful annotations:

`/**`
` * @wbp.parser.constructor`
` */`

`// $hide>>$`
`// $hide<<$`

**Comment by wimjongman** [comment]
They are all in the documentation and the code parsing preference page.

https://help.eclipse.org/latest/index.jsp?topic=/org.eclipse.wb.doc.user/html/preferences/index.html

**Reply by kl0ck** [reply]
> They are all in the documentation and the code parsing preference page.
> 
> https://help.eclipse.org/latest/index.jsp?topic=/org.eclipse.wb.doc.user/html/preferences/index.html

You are right about **_hide_** comments. But I didn't find any help about **_@wbp_** annotations.

**Reply by kl0ck** [reply]
Here is a (rare) blog post where I found some help about the annotations: 
[https://zauner.nllk.net/post/0010-eclipse-windowbuilder-javadoc-tags/](https://zauner.nllk.net/post/0010-eclipse-windowbuilder-javadoc-tags/)

**Reply by wimjongman** [reply]
I searched Eclipse help: https://help.eclipse.org

https://help.eclipse.org/latest/topic/org.eclipse.wb.doc.user/html/features/factories.html
